baybe/_optional/fpsample.py

```diff
@@ -0,0 +1,10 @@
+"""Optional fpsample imports."""
+
+from baybe.exceptions import OptionalImportError
+
+try:
+    from fpsample import fps_sampling
+except ModuleNotFoundError as ex:
+    raise OptionalImportError(name="fpsample", group="sampling") from ex
```

Comment:
if you make the suggested change to `pyproject.toml` this here needs to be adjusted accordingly

```suggestion
    raise OptionalImportError(name="fpsample") from ex
```